### PR TITLE
(greek_conversion) tlg0363.tlg007.perseus-grc2

### LINKED ISSUE #1505: (tlg0363.tlg007.perseus-grc1.xml) EpiDoc and CTS conversion
file conversion work

### PR DESCRIPTION
**Pull request by lcerrato**
fix #1505 and minor edit to tlg0284.tlg056 metadata

### CODE REVIEW

File: data/tlg0284/tlg056/__cts__.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
   
   <ti:edition urn="urn:cts:greekLit:tlg0284.tlg056.perseus-grc2" workUrn="urn:cts:greekLit:tlg0284.tlg056" xml:lang="grc">
     <ti:label xml:lang="grc">Περὶ πολιτικοῦ λόγου</ti:label>
-    <ti:description xml:lang="eng">Aristides. Rhetores Graeci, Vol. 2. Leonhard von Spengel, Leonhard. Leipzig: Teubner, 1854.</ti:description>
+    <ti:description xml:lang="eng">Aristides. Rhetores Graeci, Vol. 2. von Spengel, Leonhard, editor. Leipzig: Teubner, 1854.</ti:description>
```
Comment (AlisonBabeu) [inline data/tlg0284/tlg056/__cts__.xml:6]: title xml:lang should be "mul"

File: data/tlg0363/__cts__.xml
Comment (AlisonBabeu) [inline data/tlg0363/__cts__.xml:1]: I'm wondering if we should change this, I know we have Claudius Ptolemy here, Cladius Ptolemaeus in Scaife, and Ptolemy, 2nd century in the catalog. I'm thinking we should go with Ptolemy, 2nd cent, but then remembering we don't tend to include dates in author names.

**Review by AlisonBabeu — CHANGES_REQUESTED** [review]
hi @lcerrato just one or two possible changes, otherwise it looks good!

File: data/tlg0363/__cts__.xml
Comment (lcerrato) [reply data/tlg0363/__cts__.xml:1]: I think Ptolemy is the wiki version. Loeb and TLG add the Claudius in some form.

Comment (lcerrato) [reply data/tlg0363/__cts__.xml:1]: I think Egypt when I see just plain Ptolemy, but it would simplify matters.

**Review by AlisonBabeu — APPROVED** [review]
Looks good.

**Comment by lcerrato** [comment]
@AlisonBabeu So I don't typically check OGL for other metadata and you are right in that we now have two different workgroup names for tlg0363. Should I merge this and then you can decide which one to regularize (or just change both)? Whatever is easiest for you.

**Comment by AlisonBabeu** [comment]
hi @lcerrato go ahead and merge at this point. I'm really not sure what name to go with,if only there was a catalog with variant names....

**Comment by lcerrato** [comment]
@AlisonBabeu 
I think the issue here is this will duplicate Ptolemy in the Scaife Viewer TOC, so either I should change this to match OGL or change OGL to match this.

**Comment by AlisonBabeu** [comment]
Hmm, @lcerrato can you change this to match OGL I think...

**Pull request merged**